### PR TITLE
Authentification backend

### PR DESCRIPTION
**Pull request by Eowiin**

### CODE REVIEW

File: backend/playfit/authentification/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CustomUser',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('email', models.EmailField(max_length=254, unique=True)),
+                ('username', models.CharField(max_length=150, unique=True)),
+                ('password', models.CharField(max_length=150)),
```
[the remark this answers is not in the document]
Comment (Eowiin) [reply backend/playfit/authentification/migrations/0001_initial.py:21]: Automatically encrypted using set_password function

File: backend/playfit/authentification/migrations/__init__.py
[the remark this answers is not in the document]
Comment (Eowiin) [reply backend/playfit/authentification/migrations/__init__.py:1]: Generated automatically with django

File: backend/playfit/playfit/settings.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-93e_i_@ir2h%l2n1un*!r@8uyzto!(&+ok_delroe=$qndw3p@'
```
[the remark this answers is not in the document]
Comment (Eowiin) [reply backend/playfit/playfit/settings.py]: I moved it to environment

File: backend/playfit/authentification/migrations/__init__.py
[the remark this answers is not in the document]
Comment (Eowiin) [reply backend/playfit/authentification/migrations/__init__.py:1]: It marks a directory as python package to import its modules from other files

**Review by christophechr — APPROVED** [review]

**Review by Masteruuuuu — APPROVED** [review]
Great way to import libs to not charge the app

**Pull request merged**